soc/arm/atmel_sam0/same51/soc.h

```diff
@@ -32,6 +32,8 @@
 
 #endif /* _ASMLANGUAGE */
 
+#include "adc_fixup.h"
+
 #include "sercom_fixup_samd5x.h"
```

Comment:
I will suggest to change fixup name to `adc_fixup_sam0.h` This make explicit this is only related to sam0 platform.
The fixup `#include` line should be together with other `#includes`. In this case, drop line 36.
Please, review same with all other soc.h files.

Reply:
This is in the `soc/arm/atmel_sam0/...` tree, so hopefully we don't need that in the header too?

Reply:
It should follow current standard file name `<driver>_fixup_<platform/soc>.h`